banyan-utils/examples/filter_tiny.rs

```diff
@@ -0,0 +1,102 @@
+use std::{
```

Comment:
This example is just so I can run it through flamegraph. I guess it does not hurt to leave it in there, but I can also remove it again.

Reply:
:+1: for remove or for keep???

Reply:
keep

Reply:
Your above comment would be nice to persist in the file.

Reply:
will do.